src/main/java/org/phyloref/jphyloref/JPhyloRef.java

```diff
@@ -13,27 +13,31 @@
 import org.phyloref.jphyloref.commands.TestCommand;
 
 /**
- * Main class for JPhyloRef. Figures out what the user 
- * wants us to do. 
+ * Main class for JPhyloRef. Contains a list of Commands,
+ * as well as the code for determining which Command to
+ * execute. 
  *
  */
 public class JPhyloRef 
 {
+	/** Version of JPhyloRef */
 	public static final String VERSION = "0.0.1-SNAPSHOT";
 	
+	/** List of all commands included in JPhyloRef */
 	private List<Command> commands = Arrays.asList(
 		new HelpCommand(),
 		new TestCommand(),
 		new ReasonCommand()
 	);
 	
-    public static void main( String[] args )
-    {
-        JPhyloRef jphyloref = new JPhyloRef();
-        jphyloref.execute(args);
-    }
-    
+	/**
+	 * Interpret the command line arguments to determine which command
+	 * to execute.
+	 * 
+	 * @param args Command line arguments
+	 */
     public void execute(String[] args) {
+    	// Display version information.
     	System.err.println("jphyloref/" + VERSION + "\n");
```

Comment:
It seems bad practice to print something to STDERR without control by the user, and not signifying any error or potential error condition.